### ISSUE TITLE
Stop and spew a resonable error message when no tests are found

### ISSUE THREAD

**Issue by vrutkovs** [Containers-Testing-Framework/ctf-cli#9]
Running ctf-cli from a dir with no tests gives a mysterious message:

```
./ctf-cli.py -c ctf.conf.sample
INFO: Running Containers Testing Framework cli
WARNING: Not using project specific Features. '/home/dbecvari/repository/github/l-d-j/ctf-cli/test/features' does not exist!
WARNING: Not using project specific Steps. '/home/dbecvari/repository/github/l-d-j/ctf-cli/test/steps' does not exist!
WARNING: Not using project specific environment.py. '/home/dbecvari/repository/github/l-d-j/ctf-cli/test/environment.py' does not exist!
INFO: Running behave inside working directory 'behave -D DOCKERFILE=/home/dbecvari/repository/github/l-d-j/ctf-cli/Dockerfile -D ANSIBLE=/home/dbecvari/repository/github/l-d-j/ctf-cli/ctf-cli-behave-working-dir/ansible.conf'
usage: behave [options] [ [DIR|FILE|FILE:LINE] ]+
behave: error: unrecognized arguments: -D -D ANSIBLE=/home/dbecvari/repository/github/l-d-j/ctf-cli/ctf-cli-behave-working-dir/ansible.conf 
```

We should be more clear about why it has failed and give a usage output

CC @l-d-j

**Comment by rwngwn** [comment]
actually it seems that it was caused by older version of behave (in fedora rpm there is 1.2.4 but -D option is only present in 1.2.5 version as you can see here:
https://github.com/behave/behave/compare/v1.2.4...v1.2.5#diff-05dbd6d78643bb1dc9fcb840ad567931R86 )

**Comment by vrutkovs** [comment]
> actually it seems that it was caused by older version of behave

Yes, that also is an issue, however it will be solved by #8

**Comment by vrutkovs** [comment]
Much better now:

```
$ ctf-cli run
INFO:   Running Containers Testing Framework cli
WARNING:    Not using project specific Features. '/tmp/features' does not exist!
WARNING:    Not using project specific Steps. '/tmp/steps' does not exist!
WARNING:    Not using project specific environment.py. '/tmp/environment.py' does not exist!
INFO:   Running behave inside working directory 'behave -D DOCKERFILE=/tmp/Dockerfile -D ANSIBLE=/tmp/workdir/ansible.conf'
ConfigError: No feature files in "/tmp/workdir"
```
